wagtail_localize/models.py

```diff
@@ -126,6 +127,15 @@ def get_edit_url(instance):
             ],
         )
 
+    else:
+        return reverse(
```

Comment:
note: this implies one does use modeladmin which won't necessarily be the case.

My take on this is: `wagtail_localize.modeladmin` should be optional. If not added to `INSTALLED_APPS` then this the `SubmitModelAdminTranslationView` vuew and `submit_modeladmin_translation` path declaration should not register and come into play at all